core/controllers/acl_decorators_test.py

```diff
@@ -58,29 +59,29 @@ class OpenAccessDecoratorTests(test_utils.GenericTestBase):
 
     class MockHandler(base.BaseHandler):
         GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
-        URL_PATH_ARGS_SCHEMAS = {}
-        HANDLER_ARGS_SCHEMAS = {'GET': {}}
+        URL_PATH_ARGS_SCHEMAS: Dict[str, str] = {}
+        HANDLER_ARGS_SCHEMAS: Dict[str, Dict[str, str]] = {'GET': {}}
 
         @acl_decorators.open_access
-        def get(self):
-            return self.render_json({'success': True})
+        def get(self) -> None:
+            self.render_json({'success': True})
```

Comment:
In most places `return` is not used with `self.render_json` because `self.render_json` does not return anything and using return with `self.render_json` feels just redundant.